build.gradle

```diff
@@ -4,16 +4,18 @@ buildscript {
     ext.kotlin_version = '1.3.50'
 
     repositories {
-        mavenLocal()
         google()
         jcenter()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.2'
+        classpath 'com.android.tools.build:gradle:3.6.1'
```

Comment:
Can you check if things are still working with 3.5.3? That's the newest version still supported in Android Studio 3.5. If not, we'll just update

Reply:
I do not know, how to downgrade Android Studio plugin to 3.5 to test it :( but this change was probably done because of some warning/note popup from latest AndroidStudio, so that is why I have changed it ... could you please update you IDE 

> I guess we don't want to force our customers to update their dev tools, as it has been working till now. @Tigraine ?

This change affects only us developers, customers used built versions of our collector library

Reply:
Ah so in Android Studio >= 3.6, old gradle versions aren't working? Didn't know that, will update then :)

Reply:
let me know once you update Android Studio (and plugins, as it is necessary as well, most of the time when IntelliJ is updated) how it works by you,  I will wait with merge for you approve.